src/tr/merlinscans/build.gradle

```
@@ -1,9 +1,8 @@
 ext {
     extName = 'Merlin Scans'
     extClass = '.MerlinScans'
-    themePkg = 'mangathemesia'
     baseUrl = 'https://merlinscans.com'
-    overrideVersionCode = 1
+    extVersionCode = 2
```

Current ext version is 31
```suggestion
    extVersionCode = 32
```